mongoid.js

```diff
@@ -44,12 +44,12 @@ function MongoId( machineId ) {
     if (!machineId) machineId = Math.floor(Math.random() * 0x1000000);
     else if (machineId < 0 || machineId > 0x1000000)
         throw new Error("machine id out of range 0.." + parseInt(0x1000000));
-
-    this.processIdStr = this.hexFormat(machineId, 6) + this.hexFormat(process.pid, 4);
+    if (!process.pid) process.pid = 20000;
```

Comment:
1. is the value "20000" special in browserify, or can it be anything?  It might be better to pick a random value instead of every id factory using the same constant.
2. we probably don't want to change the global `process` object, other packages might also be using the `pid` to test whether it's nodejs or browser